.travis.yml

```diff
@@ -42,6 +42,7 @@ before_install:
   - edm install -y wheel click coverage
 install:
   - for toolkit in ${TOOLKITS}; do edm run -- python etstool.py install --runtime=${RUNTIME} --toolkit=${toolkit} || exit; done
+  - if [[ ${TRAVIS_EVENT_TYPE} == 'cron' ]]; then for toolkit in ${TOOLKITS}; do edm run -- python etstool.py install --runtime=${RUNTIME} --toolkit=${toolkit} --source || exit; done; fi
```

Comment:
I only realized this later that this causes `install` to be called twice on the cronjob.
Options:
(1) in `etstool.py`, set the default of `--source` based on some environment variable, e.g. `default=os.environ.get("IS_CRON", False)`
(2) compose the option flags in the `travis.yml` based on the environment variable..., e.g. `if [[${TRAVIS_EVENT_TYPE} == 'cron' ]] FLAGS="--source"` something like that.

Reply:
yes. sorry. i wasn't trying to optimize CI time. I'll look into this this week (before i start adding more cron jobs which are wasteful).